t/180_verify_SSL.t

```diff
@@ -0,0 +1,108 @@
+#!perl
+
+use strict;
+use warnings;
+use Test::More 0.88;
+use lib 't';
+
+use HTTP::Tiny;
+
```

Comment:
This test should delete `PERL_HTTP_TINY_INSECURE_BY_DEFAULT` from the environment to start.

Reply:
Thanks, this is fixed in both `180_verify_SSL.t` and `210_live_ssl.t`